db/migrate/20170503172351_add_web_domain_to_accounts.rb

```diff
@@ -0,0 +1,16 @@
+class AddWebDomainToAccounts < ActiveRecord::Migration[5.0]
+  def up
+    add_column :accounts, :web_domain, :string, null: true, default: nil
+
+    ActiveRecord::Base.transaction do
```

Comment:
I don't think that would ever take days, but hours is entirely possible, and that's not acceptable for a migration script. For the record, on my really small instance (only ~2k remote accounts known, but also low-end ARM hardware), the migration script took about 2 minutes to run.